com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -167,36 +162,30 @@ public BitSerializer __beginSendClientRpc(ClientRpcParams clientRpcParams, RpcDe
                     rpcQueueContainer.SetLoopBackFrameItem(clientRpcParams.Send.UpdateStage);
 
                     //Switch to the outbound queue
-                    writer = rpcQueueContainer.BeginAddQueueItemToFrame(RpcQueueContainer.QueueItemType.ClientRpc, Time.realtimeSinceStartup, Channel.ReliableRPC, 0, NetworkId,
+                    writer = rpcQueueContainer.BeginAddQueueItemToFrame(RpcQueueContainer.QueueItemType.ClientRpc, Time.realtimeSinceStartup, Channel.ReliableRpc, NetworkId,
                         ClientIds, QueueHistoryFrame.QueueFrameType.Outbound, NetworkUpdateStage.PostLateUpdate);
 
                     if (!isUsingBatching)
                     {
-                        writer.WriteBit(false); // Encrypted
-                        writer.WriteBit(false); // Authenticated
-                        writer.WriteBits(MLAPIConstants.MLAPI_CLIENT_RPC, 6); // MessageType
+                        writer.WriteByte(MLAPIConstants.MLAPI_CLIENT_RPC); // MessageType
```

Comment:
can this change go into a separate PR?

can this PR work with this: `writer.WriteBits(MLAPIConstants.MLAPI_CLIENT_RPC, 6); // MessageType`
instead of this change: `writer.WriteByte(MLAPIConstants.MLAPI_CLIENT_RPC); // MessageType`